packages/cadl-rlc-test/package.json

```diff
@@ -7,7 +7,8 @@
     "@cadl-lang/ts-rlc": "0.0.1",
```

Comment:
I think we have a decision to change the name of the emitter.

Reply:
@joheredi  Do you have any chance to confirm the name `@azure-tools/cadl-typescript` with Brian?
https://github.com/Azure/cadl-azure/issues/1759#issuecomment-1201857743